test/e2e/network/dns.go

```diff
@@ -449,7 +449,7 @@ var _ = SIGDescribe("DNS", func() {
 		By("changing the service to type=ClusterIP")
 		_, err = framework.UpdateService(f.ClientSet, f.Namespace.Name, serviceName, func(s *v1.Service) {
 			s.Spec.Type = v1.ServiceTypeClusterIP
-			s.Spec.ClusterIP = "10.0.0.123"
+			s.Spec.ClusterIP = "10.0.0.111"
```

Comment:
There needs to at least be a TODO comment to say that this is hardcoded and must lie within the service range.

Why is this changed from x.x.x.123 to .111?

Reply:
I didn't know we can partially cherry-pick a PR when I created this PR. I was just thinking to create a seed PR for chrry-picking, so I change the ClusterIP to be another randomly valid IP :)